src/response/records.ts

```diff
@@ -361,6 +361,111 @@ export interface Disposition {
   readonly ruleLabel?: string;
 }
 
+/**
+ * The risk score reason for the multiplier.
+ *
+ * This class provides both a machine-readable code and a human-readable
+ * explanation of the reason for the risk score, see
```

Comment:
This is a comma splice. It would probably be better as:

```suggestion
 * explanation of the reason for the risk score. See
```